src/steps/check-lead-activity.ts

```diff
@@ -29,6 +29,29 @@ export class CheckLeadActivityStep extends BaseStep implements StepInterface {
     optionality: FieldDefinition.Optionality.OPTIONAL,
   }];
 
+  protected expectedRecords: ExpectedRecord[] = [{
+    id: 'matchedActivities',
+    type: RecordDefinition.Type.TABLE,
```

Comment:
This may be a limitation of the API, but the data that comes back here is...  Also kinda 🙁.  Very technical-looking and not particularly useful.

![Screen Shot 2020-02-12 at 6 35 19 PM](https://user-images.githubusercontent.com/3496491/74396305-73ce8880-4dc6-11ea-814c-78bd2c0eea8f.png)

Ideally, we could mimic the experience of the Lead Activity log in Marketo, e.g.:

![Screen Shot 2020-02-12 at 6 37 13 PM](https://user-images.githubusercontent.com/3496491/74396438-d3c52f00-4dc6-11ea-89b5-aa30e5bb49f1.png)

Are there any more fields or details we could pull from the `getActivityPagingToken()` that would apply to most/all activity types?